### PR TITLE
gamin:change the SOURCE URL

### PR DESCRIPTION
**Pull request by lhw828**

### CODE REVIEW

**Comment by lunar-linux-b01** [comment]
SUCCESS!

File: utils/gamin/DETAILS [hunk 1]
```diff
@@ -1,7 +1,7 @@
           MODULE=gamin
          VERSION=0.1.10
           SOURCE=$MODULE-$VERSION.tar.gz
-      SOURCE_URL=https://web.archive.org/web/20141226011517/https://people.gnome.org/~veillard/gamin/sources/
+      SOURCE_URL=https://mirrors.ustc.edu.cn/gnome/sources/gamin/0.1/
```
Comment (Ratler) [inline utils/gamin/DETAILS:4]: We need to find a better mirror than from China.

**Review by Ratler — CHANGES_REQUESTED** [review]

**Comment by lunar-linux-b01** [comment]
SUCCESS!

**Comment by stumbles** [comment]
We don't accept a China link.

Use this;

https://salsa.debian.org/gnome-team/gamin/-/archive/upstream/0.1.10/gamin-upstream-0.1.10.tar.bz2

**Comment by stumbles** [comment]
All patches apply fine here so don't know why CI would bork on them.

**Pull request merged**